ingestion-beam/src/main/java/com/mozilla/telemetry/SimpleSink.java

```diff
@@ -0,0 +1,89 @@
+package com.mozilla.telemetry;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.mozilla.telemetry.decoder.DecoderOptions;
+import com.mozilla.telemetry.options.SinkOptions;
+import com.mozilla.telemetry.util.Json;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Base64;
+import java.util.zip.GZIPInputStream;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.MapElements;
+import org.apache.beam.sdk.values.TypeDescriptors;
+import org.apache.commons.compress.utils.IOUtils;
+
+public class SimpleSink {
+
+  /**
+   * Execute an Apache Beam pipeline.
+   *
+   * @param args command line arguments
+   */
+  public static void main(String[] args) {
+    run(args);
+  }
+
+  /**
+   * Execute an Apache Beam pipeline and return the {@code PipelineResult}.
+   *
+   * @param args command line arguments
+   */
+  public static PipelineResult run(String[] args) {
+    registerOptions();
+    final SinkOptions.Parsed options = SinkOptions.parseSinkOptions(
+        PipelineOptionsFactory.fromArgs(args).withValidation().as(SinkOptions.class));
+
+    return run(options);
+  }
+
+  /**
+   * Execute an Apache Beam pipeline and return the {@code PipelineResult}.
+   */
+  public static PipelineResult run(SinkOptions.Parsed options) {
+    final Pipeline pipeline = Pipeline.create(options);
+
+    pipeline //
+        .apply(options.getInputType().read(options)) //
```

Comment:
This line ends up calling BQ direct reads. See

https://github.com/mozilla/gcp-ingestion/blob/case-22698806/ingestion-beam/src/main/java/com/mozilla/telemetry/io/Read.java#L95-L165